libraries/datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -348,6 +349,29 @@ public static String printFullname(User user) {
         return sb.toString();
     }
 
+    public static boolean isUserAtleastDesiredRoleInPrimaryOrSecondaryGroup(User user, UserGroup userGroup) {
+        if (PermissionUtils.isUserAtLeast(userGroup, user)) {
+            return true;
+        }
+        if (CommonUtils.isNullOrEmptyMap(user.getSecondaryDepartmentsAndRoles())) {
+            return false;
+        }
+        return (PermissionUtils.isUserAtLeastDesiredRoleInSecondaryGroup(userGroup,
+                user.getSecondaryDepartmentsAndRoles().values().stream().flatMap(Set::stream).collect(Collectors.toSet())));
+    }
+
+    public static String getSW360Version() {
+        MavenXpp3Reader reader = new MavenXpp3Reader();
+        try (InputStreamReader iStreamReader = new InputStreamReader(
+                SW360Utils.class.getResourceAsStream("/META-INF/maven/org.eclipse.sw360/datahandler/pom.xml"))) {
```

Comment:
Kindly try to remove the hard-coded path

Reply:
I have made in configurable via `sw360.properties` : `datahandler.pom.file.path`